README.Rmd

```diff
@@ -42,73 +42,67 @@ knitr::opts_chunk$set(
 [![rstudio mirror downloads](https://cranlogs.r-pkg.org/badges/rredlist)](https://github.com/metacran/cranlogs.app)
 [![cran version](https://www.r-pkg.org/badges/version/rredlist)](https://cran.r-project.org/package=rredlist)
 
-`rredlist` is an R client for the IUCN Red List (http://apiv3.iucnredlist.org/api/v3/docs). 
+`rredlist` is an R client for the IUCN Red List (<https://apiv3.iucnredlist.org/api/v3/docs>). 
 The IUCN Red List is a global list of threatened and endangered species.
 
-IUCN Red List docs: http://apiv3.iucnredlist.org/api/v3/docs
```

Comment:
I prefer to minimize linkable URLs in README's because if they are down at all during a cran check then it may make a submission take another round. And anyway urls as strings on github and docs pages show up as clickable links

Reply:
ok, removing them